### ISSUE TITLE
[Core] Feature request: ability to list all changes from a given Era

### ISSUE THREAD

**Issue by kpedro88** [cms-sw/cmssw#14840]
With Phase1 and Phase2 development ongoing, the number of Eras and associated configuration changes is increasing dramatically.

Right now the only way to identify all the changes from a given Era is searching on GitHub, using grep, etc.

It would be nice to add a feature to the framework which would keep track of all the changes applied by each Era, so that this information could be collected and printed on demand (perhaps given a certain configuration file).

At minimum, it seems like this would require changes to the Modifier and ModifierChain Python objects. Maybe a new edm utility would also be helpful. Open to ideas from framework experts...

**Comment by Dr15Jones** [comment]
assign core

**Comment by boudoul** [comment]
Hi all , I have no idea whether this requested feature is complicated or not to implement, but It would certainly help a lot

**Comment by kpedro88** [comment]
@davidlange6 it seems like #15480 makes some progress toward this, but something is still missing. When I enable `eras.inspect()` in `Configuration/StandardSequences/python/Eras.py`, it prints things like this:
```
Era: Run2_2017
   isChosen: False
   Modifiers:
       []
       []
       []
       []
       []
    5 modifiers defined
```

**Comment by davidlange6** [comment]
no, its not done - but in any case, it will require a full config to get sensible answers.

**Comment by smuzaffar** [comment]
@davidlange6 and @kpedro88 , can we close this now?

**Comment by kpedro88** [comment]
@smuzaffar no, this is not actually implemented yet.
